*** src/fe/fe.C.orig	2016-02-03 09:55:25.247604001 +0100
--- src/fe/fe.C	2016-02-03 09:59:21.089126671 +0100
*************** void FE<Dim,T>::init_shape_functions(con
*** 297,303 ****
          }
      }
  #else
!   if (!this->calculate_phi && !this->calculate_dphi && !this->calculate_curl_phi && !this->calculate_div_phi)
      {
        this->calculate_phi = this->calculate_dphi = this->calculate_dphiref = true;
        if( FEInterface::field_type(T) == TYPE_VECTOR )
--- 297,309 ----
          }
      }
  #else
!   // this->calculate_... is set false by default and changed to true, if I asked for it (->get_...() )
!   // in the assembly-function. 
!   // Later,  I calculate dphid... if calculate_dphiref is true -> I need to set it when ever
!   // calculate_dphi is requested, I think. 
!   // So the expression should be true whenever at least one ef the calculate_... is true:
!   //if (this->calculate_phi || this->calculate_dphi || this->calculate_curl_phi || this->calculate_div_phi)  // equivalent to the following statement:
!   if (!(!this->calculate_phi && !this->calculate_dphi && !this->calculate_curl_phi && !this->calculate_div_phi))
      {
        this->calculate_phi = this->calculate_dphi = this->calculate_dphiref = true;
        if( FEInterface::field_type(T) == TYPE_VECTOR )
