*** src/fe/fe.C.orig	2016-02-18 08:46:44.228117552 +0100
--- src/fe/fe.C	2016-02-18 08:46:49.680021884 +0100
*************** void FE<Dim,T>::init_base_shape_function
*** 571,577 ****
  {
    // I don't understand infinite elements well enough to risk
    // calculating too little.  :-(  RHS
!   this->calculate_phi = this->calculate_dphi = this->calculate_d2phi = true;
  
    this->elem_type = e->type();
    this->_fe_map->template init_reference_to_physical_map<Dim>(qp, e);
--- 571,577 ----
  {
    // I don't understand infinite elements well enough to risk
    // calculating too little.  :-(  RHS
!   this->calculate_phi = this->calculate_dphi = this->calculate_dphiref = true;
  
    this->elem_type = e->type();
    this->_fe_map->template init_reference_to_physical_map<Dim>(qp, e);
