*** src/mesh/mesh_generation.C	(original)
--- src/mesh/mesh_generation.C	(new version)
***************
*** 1518,1527 ****
    libmesh_assert_greater (rad, 0.);
    //libmesh_assert_greater (nr, 0); // must refine at least once otherwise will end up with a square/cube
  
    START_LOG("build_sphere()", "MeshTools::Generation");
  
    // Clear the mesh and start from scratch
    mesh.clear();
  
    BoundaryInfo& boundary_info = mesh.get_boundary_info();
  
--- 1518,1528 ----
    libmesh_assert_greater (rad, 0.);
    //libmesh_assert_greater (nr, 0); // must refine at least once otherwise will end up with a square/cube
  
    START_LOG("build_sphere()", "MeshTools::Generation");
  
    // Clear the mesh and start from scratch
+   unsigned int dim = mesh.mesh_dimension();
    mesh.clear();
+   mesh.set_mesh_dimension(dim);
  
    BoundaryInfo& boundary_info = mesh.get_boundary_info();
  
