--- src/esp/esp.F       2016-03-16 19:17:08.000000000 +0100
+++ src/esp/esp.F       2016-11-17 15:46:00.006268472 +0100
@@ -65,7 +65,9 @@ c
 c
 c     calculate the electrostatic potential on the grid
 c
-      if(newgrd.ne.0.or.denmat.ne.0) then 
+c     -> call esp_esp anyway (Hubert)
+c      if(denmat.ne.0) then 
+c      if(newgrd.ne.0.or.denmat.ne.0) then 
          call esp_esp(dbl_mb(ieptr),  !ene
      +                dbl_mb(ioptr),  !occ 
      +                dbl_mb(idptr),  !dns
@@ -73,7 +75,7 @@ c
      +                dbl_mb(iqptr),  !q
      +                dbl_mb(igptr),  !grid
      +                dbl_mb(ivptr))  !val
-      end if ! newgrd, denmat
+c      end if ! newgrd, denmat
 c
 c     do the fitting
 c

